### DISCUSSION TITLE
Errors fetching aptos framework dep

### DISCUSSION THREAD

**Discussion by LawsonGraham** [aptos-labs/aptos-developer-discussions#325] [Questions] [answered]
### Describe your question in detail.

I get this error:

```
aptos move test                                                
{
  "Error": "Unexpected error: Failed to run tests: Unable to resolve packages for package 'Test': While resolving dependency 'AptosFramework' in package 'Test': Failed to reset to latest Git state 'mainnet' for package 'AptosFramework', to skip set --skip-fetch-latest-git-deps | Exit status: exit status: 128"
}
```
on any package I run `aptos move test` in, and have to skip fetching deps to get around it, however now I actually do need to fetch latest deps so I am stuck on how to run tests now. 

### Which operating system are you using?

macOS

### Which SDK or tool are you using? (if any)

Aptos CLI

**Comment by 0x-j** [comment]
can you share source code? i want to try running your code

**Comment by teenu4** [comment, marked as answer]
Hello
Did you check this comment? Solved the issue for me https://github.com/aptos-labs/aptos-developer-discussions/discussions/282#discussioncomment-9884427

**Reply by LawsonGraham** [reply]
this was it - thanks!

**Reply by MoonShiesty** [reply]
🫡
